src/Domains/Connectors/RainForest/Enums/RainForestEnum.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Connectors\RainForest\Enums;
+
+enum RainForestEnum: string
```

Comment:
@FredPeal check the name we use inthe other connection around ConfigurationEnum and CustomFieldEnum